README.md

```diff
@@ -7,67 +7,98 @@ Perses
 
 ## Overview
 
-Perses is part of the [CoreDash community](https://github.com/coredashio/community).
+Perses is part of the [CoreDash community](https://github.com/coredashio/community). It belongs to the Linux Foundation.
+On a later stage, we want to promote the project to the [Cloud Native Computing Foundation](https://www.cncf.io/) and be
+part of the monitoring tools like Prometheus or Thanos.
+
+Perses is going to tackle multiple different goals:
+
+1. It aims to become a **standard** dashboard visualization tool for Prometheus and other datasources. It will focus on
+   being GitOps-compatible and thus enabling a smooth "dashboards as code" workflow via a new and well-defined dashboard
+   definition model.
+2. While becoming another visualization tool, Perses also aims to provide different npm packages, so it can benefit to
+   anyone that would like to integrate data visualization in his UI. For example, these packages might be used to
```

Comment:
Maybe something like "so it can benefit anyone that would like to embed charts and dashboards in their UI."